### PR TITLE
added some experimental_functions

### PR DESCRIPTION
**Pull request by vandit98**
allclose #11608
imag #11607

### CODE REVIEW

**Review by MahmoudAshraf97 — COMMENTED** [review]
imag and allclose are good to merge, please review the others and make sure they pass the tests

File: ivy/functional/backends/paddle/experimental/elementwise.py [hunk 1]
```diff
@@ -245,7 +251,7 @@ def fix(
     *,
     out: Optional[paddle.Tensor] = None,
 ) -> paddle.Tensor:
-    raise IvyNotImplementedException()
+    paddle.floor(x, name="fix")
```
Comment (MahmoudAshraf97) [inline ivy/functional/backends/paddle/experimental/elementwise.py:254]: AssertionError: The length of results from backend paddle and ground truth framework tensorflow does not match

**Review by MahmoudAshraf97 — CHANGES_REQUESTED** [review]

**Comment by vandit98** [comment]
hi @MahmoudAshraf97 
How can i make sure that my function are passing test. Can u please elaborate how to do do testing.
What tests my functions should pass? 
So far i tried passing my own inputs.
thanks

**Comment by MahmoudAshraf97** [comment]
use this command in terminal and make sure you are in the right directory
`pytest -B paddle ivy_tests/test_ivy/test_functional/test_experimental/test_core/test_elementwise.py::test_function_name`

**Comment by vandit98** [comment]
thanks @MahmoudAshraf97

**Comment by MahmoudAshraf97** [comment]
can you separate imag and allclose in a separate PR so we can merge them? also in the next PRs make sure to add 1 function only

**Review by MahmoudAshraf97 — APPROVED** [review]

**Comment by vandit98** [comment]
hi @MahmoudAshraf97 
Can u help me with this error. I am trying to use an alreading working function.
Screenshot from 2023-03-04 18-34-58

**Comment by vandit98** [comment]
hi @MahmoudAshraf97 
Can u help me with this error. I have tried to run test for allclose.
Is there any documents or resources for doing this. 
Screenshot from 2023-03-05 01-04-13

**Pull request merged**